content/en/docs/concepts/collision-prevention.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Collision Prevention
+weight: 350
+images: []
+mermaid: true
+---
+
+When updating objects on a Kubernetes cluster, multiple actors may start to update the same object. In the worst case, this may cause kube-apiserver unavailability as multiple controllers or other automation produce high load on both the apiserver and etcd.
+
+To prevent this situation, Package Operator is by default refusing to operate on objects that have not been created by Package Operator itself.
+
+## Disable Collision Protection
+
+Package Operator offers 3 different protection levels that can be set on the object-level. This allows to disable collision protection when selected objects are already present and should be "adopted" by Package Operator.
+
+- **Prevent**  
+  The default setting.  
+  Prevents any operation on objects not created by PKO.
+- **IfNoController**  
+  Allows operations on objects without an owner reference. This includes objects created directly by users, but excludes objects created via controllers.
+- **None**  
+  Disables all collision prevention checks.  
+  *Only use this settings when you know what you are doing!*
+
+### Package
+
+To change the collision protection settings for objects within a package, use the: 
+`package-operator.run/collision-protection` annotation.
```

Comment:
An example of the `oc annotate --overwrite` command might be useful here

Reply:
Expanded the sentence to clarify, the annotation is not set on an object in the cluster, it's set within the Package filesystem. The oc command won't work here.